mxfusion/components/functions/operators/operators.py

```diff
@@ -6,22 +6,33 @@ class Operator(FunctionEvaluation):
     Abstract Operator object for using MXNet operators in MXFusion space.
     Child classes implement the eval method with their operator and access necessary state through the properties dictionary.
     """
-    def __init__(self, inputs, outputs, operator_name, properties=None):
+    def __init__(self, inputs, outputs, operator_name, properties=None, broadcastable=False):
 
         # TODO Add a flag for broadcastable
 
         input_names = [v[0] for v in inputs]
         output_names = [v[0] for v in outputs]
-        self.properties = properties
+        self._properties = properties
         self.name = operator_name
```

Comment:
Operator is a FunctionEvaluation, in which the attribute name corresponds to the name referred by the model. It is better to call it operator_name.